components/leaflet-maps/ClustersMap.vue

```diff
@@ -195,6 +195,35 @@
+      
+      if (noconversion.includes(name)){
+        return date; //return same date if in noconversion
+      }
+      let [day, month, year] = date.split('/');
+      day = Number(day);
+      month = Number(month)
+      year = Number(year);
+      let currentDate = new Date(year, month - 1, day);
+      let earlierDate = new Date(currentDate);
+      const cutoffDate = new Date(1700,3,1)
+
+      if (currentDate < cutoffDate) {
+        earlierDate.setDate(currentDate.getDate()-10);
+      }
+      else {
+        earlierDate.setDate(currentDate.getDate()-11);
+      }
+
+      const earlierDateString =
```

Comment:
I think it would be good to add comments to this function - particularly this bit, to explain what the code is doing as it isn't immediately clear. Thanks Ruby

Reply:
Will add comments

Reply:
I can't see the section of code I commented on now so think it has been edited. Happy to have resolved.